Update contributing docs (#874)
romefrontend/rome: CONTRIBUTING.md

cd rome
./rome --help
```

> ## Developing on Windows
> you need to use the backslash(`\`) to run any rome commands on windows in place of forwardslash(`/`)
> For example, to run help

Reviewer: ```suggestion

## Developing on Windows

You need to use the backslash (`\`) to run any `rome` command on Windows instead of the slash (`/`); Windows uses backslashes for file paths.

For example, to run help:
```

> ```

Reviewer: ```suggestion

```bash
```

.\rome --help
```

> or you can directly use `rome` without any path referencing like below:
> ```

Reviewer: ```suggestion
Or you can directly use `rome` without any path referencing like below:

```bash
```
